frontend/public/components/factory/list.tsx

```diff
@@ -144,6 +145,15 @@ const listFilters = {
     return fuzzyCaseInsensitive(str, displayName);
   },
 
+  'vm-status' : (statuses, vm) => {
```

Comment:
Please mark this commit name with sort of `[OKD-core]`

Reply:
added `[OKD-core]` prefix